test/units/cli/test_galaxy.py

```diff
@@ -24,11 +24,9 @@
 import tarfile
 import tempfile
 
-from mock import patch
-
 from ansible.compat.six import PY3
 from ansible.compat.tests import unittest
-from mock import patch, call
+from ansible.compat.tests.mock import call, patch
```

Comment:
They are but the change to use ansible.compat.tests.mock is needed in all of them... If you're ready for it to be committed, I can leave them in a separate commit when I squash.  (I'll jsut have to remember to do a merge commit in the github UI rather than a squash).